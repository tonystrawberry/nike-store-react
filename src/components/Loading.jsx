import React, { PureComponent } from 'react';

class Loading extends PureComponent {
  
  render() {
    return (
      <div></div>
    );
  }
};

export default Loading;