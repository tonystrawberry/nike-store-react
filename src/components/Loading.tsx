import React, { PureComponent } from 'react';
import LoadingGif from '../assets/loading.gif';

class Loading extends PureComponent {
  
  render() {
    return (
      <div>
        <img width="50" src={LoadingGif} alt="Loading"/>
      </div>
    );
  }
};

export default Loading;