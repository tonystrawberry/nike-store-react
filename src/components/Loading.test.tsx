import { shallow } from 'enzyme';
import React from 'react';
import Loading from './Loading';
import toJson from 'enzyme-to-json';

it ('expect to render Loading component', () => {
  expect(toJson(shallow(<Loading />))).toMatchSnapshot();
})
