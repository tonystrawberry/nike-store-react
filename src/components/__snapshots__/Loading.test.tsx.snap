// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Loading component 1`] = `<div />`;
